openmetadata-ui/src/main/resources/ui/src/components/DashboardVersion/DashboardVersion.component.tsx

```diff
@@ -240,47 +240,43 @@ const DashboardVersion: FC<DashboardVersionProp> = ({
 
   return (
     <>
-      <div data-testid="dashboard-version-container">
-        {isVersionLoading ? (
-          <Loader />
-        ) : (
-          <div
-            className={classNames('version-data')}
-            data-testid="version-data">
-            <Row gutter={[0, 12]}>
-              <Col span={24}>
-                <DataAssetsVersionHeader
-                  breadcrumbLinks={slashedDashboardName}
-                  currentVersionData={currentVersionData}
-                  deleted={deleted}
-                  displayName={displayName}
-                  ownerDisplayName={ownerDisplayName}
-                  ownerRef={ownerRef}
-                  tierDisplayName={tierDisplayName}
-                  version={version}
-                  onVersionClick={backHandler}
-                />
-              </Col>
-              <Col span={24}>
-                <Tabs
-                  data-testid="tabs"
-                  defaultActiveKey={tab ?? EntityTabs.DETAILS}
-                  items={tabItems}
-                  onChange={handleTabChange}
-                />
-              </Col>
-            </Row>
-          </div>
-        )}
+      {isVersionLoading ? (
+        <Loader />
+      ) : (
```

Comment:
lets take loader out from return statement

Reply:
Actually, the loader is not shown for the entire page content for these components. The `EntityVersionTimeLine` is still displayed while loading is true. So doesn't make sense to bring the loader out of return here.